Documentation/ApiOverview/CachingFramework/FrontendsBackends/Index.rst

```diff
@@ -328,9 +328,10 @@ Options
    :Description:
       Gzip compression level (if the :code:`compression` option is set to :code:`true`).
       The default compression level is usually sufficient.
-      - -1: Default gzip compression (recommended)
-      - 0: No compression
-      - 9: Maximum compression (costs a lot of CPU)
+
+      * `-1`: Default gzip compression (recommended)
```

Comment:
Indent lists with two spaces (by pressing tab after the star): https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/CommonPitfalls/Lists.html